client/src/components/pages/StartPage.js

```diff
@@ -1,14 +1,19 @@
 import Header from "../Header/Header";
 import LinkButton from "../Link/Link";
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Container from "../Container";
 
 function StartPage() {
+  const [value, setValue] = useState(localStorage.getItem("name") || "");
+  useEffect(() => {
+    localStorage.setItem("name", value);
+  }, [value]);
+  const onChange = (event) => setValue(event.target.value);
```

Comment:
Think about pulling the state up into `App.js`, from there you can pass it down to `HomePage` and  `StartPage` via props, this is a general pattern in react.

As for the extra quotation marks around name alsways use `JSON.stringify` when saving and `JSON.parse` when loading data from `localStorage`. 

Beware, `JSON.parse` will throw an error if it encounters something funny (it is a little bit picky about what to read). Data passed through `JSON.stringify` will alwys be parsed correctly, though. Best practive is to wrap calls to `JSON.parse` with a `try-catch`-statement.

Reply:
Thanks for review! 👍🏻 🙏🏻